app/scripts/modules/core/help/helpContents.js

```diff
@@ -135,6 +135,9 @@ module.exports = angular.module('spinnaker.core.help.contents', [])
       incoming traffic is directed through a named port (for Spinnaker, the named port is <b>http</b>).
       The mapping from named port to port number is specified per server group
       and can be configured within the server group creation dialogue under <b>Port Name Mapping</b>.`,
+    'gce.securityGroup.sourceCIDRs': 'Traffic is only allowed from sources that match one of these CIDR ranges, or one of the source tags above.',
+    'gce.securityGroup.sourceTags': 'Traffic is only allowed from sources that match one of these tags, or one of the source CIDR ranges below.',
+    'gce.securityGroup.targetTags': 'Traffic is only allowed only if the target instance has one of these tags. If no tags are provided, all targets are allowed.',
```

Comment:
Actually, feels like we should do something stronger here.
Before this change, a newly-created security group would have a tag automatically generated (there was no way to specify any target tags). Now, the target tags are optional, and the default behavior is 'allow traffic to all server groups'.
I'm thinking it might be better to add 2 (mutually-exclusive) checkboxes with something like 'auto-generate target tag' and 'allow traffic to all server groups' and have the target tags only permit zero entries if one of those options is selected. This way we take the safer approach for the user and also preserve the existing behavior of allowing a target tag to just be generated on the backend.